bean-utils-library/src/test/java/com/hotels/beans/BeanUtilsTest.java

```diff
@@ -109,6 +113,26 @@ public void testGetTransformerFunctionWorksProperly(final String testCaseDescrip
                 .forEach(i -> assertThat(actual.get(i), sameBeanAs(fromFooSimpleList.get(i))));
     }
 
+    /**
+     * Test that the transformer function is applied earlier than the default value
+     */
+    @Test
+    public void testTransformerFunctionHasHigherPriorityThanDefaultValue() {
+        //GIVEN
+        BeanUtils beanUtils = new BeanUtils();
```

Comment:
as you are going to move this test elsewhere, could you kindly use the `underTest` variable as `Transformer` instance?
as that variable is used from the other test cases too please add this at the end of your test:
```
underTest.resetFieldsTransformer();
```
```